RHEL.md

```diff
@@ -32,9 +32,4 @@ Use your distribution's package manager to install the `<!-- {{ package }} -->`
     dnf install <!-- {{ package }} -->
     ```
 
-=== "RHEL 7"
```

Comment:
Please remove the tab switcher completely (`===`) and the whole "RHEL 8 or later" thing.